src/infrastructure/storage.md

```diff
@@ -102,8 +102,44 @@ provision a new VM and copy the data to a smaller disk.
 
 # S3-compatible Object Storage
 
-Our Ceph cluster also provides an S3-compatible object storage. Contact our
-support to get you started with credentials.
+Our Ceph cluster also provides an S3-compatible object storage.
+The data in the cluster is transparently encrypted before being stored onto
+physical disks, see [](#data-at-rest-encryption) for details.
+
+S3 user management can be done in the customer portal at [my.flyingcircus.io](https://my.flyingcircus.io).
+`manager` permission is required to manage S3 users.
+After creating users, please allow up to 10 minutes for the user to be created.
+You can see the current status of the user in the portal.
+
+Secret keys of an user are only shown once. If you loose the secret key, you can rotate the key in the customer portal.
```

Comment:
```suggestion
Secret keys of a user are only shown once. If you loose the secret key, you can rotate the key in the customer portal.
```

https://english.stackexchange.com/questions/105116/is-it-a-user-or-an-user#105117